Update CSM Vulnerabilities Overview - part 1 (#26240)
DataDog/documentation: content/en/security/cloud_security_management/vulnerabilities/_index.md

With CSM Vulnerabilities, you can manage your cloud security management strategy, all in one place:

- Create a vulnerability management program, from CI/CD pipelines to production resources
- Pass compliance audits (SOC2, PCI, HIPAA, CIS, FedRamp, etc.)
- Remediate emerging vulnerabilities (0-day CVEs)

**Note**: For vulnerability management in application libraries, see [Software Composition Analysis][5]. For application code, see [Code Security][10].

## Key capabilities

Deploy using Agentless or unified Datadog Agent
: Quickly scan your entire infrastructure for vulnerabilities, either using Agentless, or by using the unified Datadog Agent you already have deployed.

Inventory cloud resources, in real-time
: Inventory container images, hosts, serverless functions, and all packages deployed in your infrastructure, in real time, and export your SBOM.

Detect vulnerabilities continuously
: Scan recent updates and newly published CVEs, across running container images, hosts, host images, and serverless, and identify vulnerable container image layers.

Prioritize exploitable vulnerabilities, using runtime observability

> : Take advantage of Datadog's security scoring, which is based on CVSS, and combining intel with CISA KEV, EPSS, and public exploit availability. With runtime observability, you can monitor production, exposure to attacks, sensitive data processing, and who has privileged access.

Reviewer: ```suggestion
: Leverage Datadog's security scoring, which is based on CVSS, by incorporating intel from CISA KEV, EPSS, and public exploit availability. With runtime observability, you can monitor production, exposure to attacks, sensitive data processing, and privileged access.
```

Reviewer: Or maybe it should be "which is based on CVSS and incorporates intel from..."?

Take advantage of guided remediation
: See which layers are impacted, get suggestions specific to each image, and action on your vulnerability lifecycle management.

Implement automation and integrations

> : Automate creation of Jira tickets, implement SLAs. Export vulnerabilities, coverage, and SBOMs using Datadog's public API.

Reviewer: ```suggestion
: Automate the creation of Jira tickets and implement SLAs. Use Datadog’s public API to export vulnerabilities, coverage, and SBOMs.
```

Explore reports
: View and monitor vulnerability data in your dashboards.

## Deployment methods

Get started with CSM Vulnerabilities and cover your infrastructure in minutes, using:

> - [Agentless scanning][11]

Reviewer: ```suggestion
- [Agentless Scanning][11]
```

> - [Unified Datadog agent][12]

Reviewer: ```suggestion
- [Unified Datadog Agent][12]
```

> You can also use both deployment methods to use the unified Datadog Agent where you already have it deployed, and Agentless elsewhere.

Reviewer: ```suggestion
You can also employ both deployment methods: use the unified Datadog Agent where it is already deployed, and Agentless elsewhere.
```

> After you've enabled it, Datadog starts scanning your resources continuously, and starts reporting prioritized vulnerabilities in your [CSM Vulnerability Explorer][1] within an hour. 

Reviewer: ```suggestion
After CSM Vulnerabilities is enabled, Datadog continuously scans your resources and starts reporting prioritized vulnerabilities in your [CSM Vulnerability Explorer][1] within an hour.
```

| Vulnerability prioritization              | Yes                                           | Yes, with runtime context      |
| Vulnerability scanning frequency          | 12 hours                                      | Real-time                      |

| Vulnerability detection scope             | Agentless                                     | Unified Datadog Agent          |
|-------------------------------------------|-----------------------------------------------|--------------------------------|
| Host and host image                       | OS packages and app packages, mapped to image | OS packages                    |
| Container image                           | OS packages and app packages, mapped to image | OS packages                    |
| Cloud provider                            | AWS, [Azure (Preview)][15]                    | AWS, Azure, GCP, on-prem, etc. |
| Operating system                          | Linux                                         | Linux, Windows                 |
| Serverless                                | AWS Lambda                                    | -                              |

> | Container registries                      | [Amazon ECR (Preview)][16]                    | -                              |

Reviewer: I'm assuming the dashes indicate a lack of support? I suggest using "Not applicable" or something along those lines.

Author: Checking other documentation for consistency, looks like we typically leave an empty space. 
Example: https://docs.datadoghq.com/agent/supported_platforms/?tab=linux

For more information on compatibility, see [CSM Vulnerabilities Hosts and Containers Compatibility][13]. If you need any assistance, see the [troubleshooting guide][14], or reach out to support@datadoghq.com.

## Continuously detect, prioritize, and remediate exploitable vulnerabilities

> The [CSM Vulnerabilities Explorer][1] helps you investigate vulnerabilities detected across  your container images, host images, running hosts, and serverless functions using filtering and grouping capabilities.

Reviewer: ```suggestion
The [CSM Vulnerabilities Explorer][1] helps you investigate vulnerabilities detected across your container images, host images, running hosts, and serverless functions using filtering and grouping capabilities.
```

> The [CSM Vulnerabilities Explorer][1] helps you investigate all vulnerabilities detected across container images, host images, running hosts, and serverless functions you own, using filtering and grouping capabilities.

Reviewer: ```suggestion
The [CSM Vulnerabilities Explorer][1] helps you investigate vulnerabilities detected across  your container images, host images, running hosts, and serverless functions using filtering and grouping capabilities.
```

> Focus on exploitable vulnerabilities first, using the Datadog Severity Score, combining the base CVSS score with many risk factors, including sensitive data, environment sensitivity, exposure to attacks, exploit availability, or threat intelligence sources.

Reviewer: ```suggestion
Focus on exploitable vulnerabilities first by leveraging the Datadog Severity Score, which combines the base CVSS score with various risk factors such as sensitive data, environment sensitivity, exposure to attacks, exploit availability, and threat intelligence sources.
```

> The [CSM Vulnerabilities Explorer][1] helps you address vulnerabilities that have fixes available, and provides guided remediation steps, helping Dev and Ops teams fix vulnerabilities faster. Triage, mute, comment, and assign vulnerabilities to manage their lifecycle.

Reviewer: ```suggestion
For vulnerabilities with available fixes, the Explorer provides guided remediation steps to assist Dev and Ops teams in resolving issues more quickly and effectively. You can also triage, mute, comment, and assign vulnerabilities to manage their lifecycle.
```

{{< img src="security/vulnerabilities/csm-vm-explorer-actionability.png" alt="The CSM Vulnerability Explorer displaying a vulnerability and the actions a user can take to remediate it" width="100%">}}

## Automation and Jira integration
Make CSM Vulnerabilities part of your daily workflow by setting up [security notification rules][17]:
- Get alerted upon detection of an exploitable vulnerability for your scope
- Automatically create Jira tickets

> - Configure SLAs to remediate vulnerabilities

Reviewer 2: Do you configure the SLAs in the notification rules? I don't see where you would do so in the UI.

Author: You're right, SLA will be part of automation pipelines (curently in preview)

{{< img src="security/vulnerabilities/csm-notifications.png" alt="The notification rule setup screen" width="100%">}}

## Tracking and reporting

> Track and report progress to stakeholders by using the [out-of-the-box CSM Vulnerabilities dashboard][18]. Customize this dashboard to your own needs by cloning and tweaking it as needed.

Reviewer: ```suggestion
Use the out-of-the-box [CSM Vulnerabilities dashboard][18] to track and report progress to stakeholders. Clone and modify it as needed to fit your unique needs.
```

{{< img src="security/vulnerabilities/csm-vm-reporting.png" alt="The CSM Vulnerabilities dashboard" width="100%">}}

## Explore infrastructure packages

> The [Infrastructure Packages Catalog][19] provides a real-time inventory of all packages across hosts, host images, and container images deployed in your infrastructure. It offers an interface you can use to investigate your SBOMs, enriched with vulnerability and runtime context.

Reviewer: This link isn't working for me, and I'm not able to find the corresponding page in the UI. Is it in Preview/hidden behind a feature flag? If it's in Preview, we should call that out.

Author: Its gonna be released today

## Overview

> Cloud Security Management Vulnerabilities (CSM Vulnerabilities) helps you improve your security posture and achieve compliance by continuously scanning container images, hosts, host images and serverless functions for vulnerabilities, from CI/CD pipelines to live production. Leveraging runtime observability, it helps you prioritize and remediate exploitable vulnerabilities in your daily workflows, making Datadog your single pane of glass.

Reviewer 2: ```suggestion
Cloud Security Management Vulnerabilities (CSM Vulnerabilities) helps you improve your security posture and achieve compliance, by continuously scanning container images, hosts, host images, and serverless functions for vulnerabilities, from CI/CD pipelines to live production. Leveraging runtime observability, it helps you prioritize and remediate exploitable vulnerabilities in your daily workflows, all in a single view, and without any dependencies on other Datadog products.
```

> **Note**: For vulnerability management in application libraries, see [Software Composition Analysis][5] and for application code, see [Code Security][10].
> 
> **Note 2**: CSM Vulnerabilities is available without any dependencies with other Datadog products.
> 
> ## Use-cases
> 
> - Vulnerability Management program, from CI/CD pipelines to production resources
> - Pass compliance audit (SOC2, PCI, HIPAA, CIS, FedRamp,..)
> - Emerging vulnerabilities (0-day CVEs)

Reviewer 2: ```suggestion
With CSM Vulnerabilities, you can manage your cloud security management strategy, all in one place:

- Create a vulnerability management program, from CI/CD pipelines to production resources
- Pass compliance audits (SOC2, PCI, HIPAA, CIS, FedRamp, etc.)
- Detect emerging vulnerabilities (0-day CVEs)

**Note**: For vulnerability management in application libraries, see [Software Composition Analysis][5]. For application code, see [Code Security][10].
```

## Key capabilities

Deployment via Agentless or unified Datadog Agent
: Quickly scan your entire infrastructure for vulnerabilities via Agentless or by using the unified Datadog Agent you already deployed.

Inventory of cloud resources, in real-time
: Real time inventory of container images, hosts, serverless functions and all packages deployed in your infrastructure. SBOM export.

Continuous vulnerability detection 
: Scan updates pushed and new CVE published. Across running container images, host, host images and serverless. Identify container image layer that is vulnerable.

Prioritization of exploitable vulnerabilities, using runtime observability
: Based on CVSS, combining intel: CISA KEV, EPSS, Public exploit availability. Runtime observability: production, exposure to attack, processing sensitive data, priviledged access.

Guided remediation
: Impacted layer, image suggestion, vulnerability lifecycle management

Automation and Integration
: Automate creation of JIRA tickets, implement SLAs. Export vulnerabilities, coverage and SBOMs via public API.

Reporting

> : OOTB dashboard

Reviewer 2: I initially found some of these these list items hard to follow, because they were lists of nouns without context. I tried to make them full sentences, adding verbs to add context that could help users understand what they could action on. Could you review these edits in particular for accuracy? I'm sure there are ways these sentences can be improved upon or clarified.

## Getting started

### Enable CSM Vulnerabilities
Get started with CSM Vulnerabilities now and cover your infrastructure in minutes, using:
- [Agentless scanning][11]
- [Unified Datadog agent][12]

> You can also use both deployment methods to leverage the unified Datadog Agent where already deployed and Agentless elsewhere.
> 
> Once enabled, your resources are continuously scanned and prioritized vulnerabilities will start being reported in your [CSM Vulnerability Explorer][1] within an hour. 

Reviewer 2: ```suggestion
You can also use both deployment methods to use the unified Datadog Agent where you already have it deployed, and Agentless elsewhere.

After you've enabled it, Datadog starts scanning your resources continuously, and starts reporting prioritized vulnerabilities in your [CSM Vulnerability Explorer][1] within an hour. 
```

> Below is the table to help you define which solution to start with: 
> | Feature                         | Agentless              | Unified Datadog Agent     |
> |---------------------------------|------------------------|---------------------------|
> | Time to deploy accross your infrastrcture       | Minutes | Hours to weeks           |
> | Vulnerability prioritization                    | Yes | Yes, with runtime context    |
> | Vulnerability scanning frequency                | 12 hours | Real-time               |
> | Vulnerability detection scope |
> | Host and Host image           | OS packages and App packages, mapped to image | OS packages |
> | Container image               | OS packages and App packages, mapped to image | OS packages |
> | High-level Compatibility - [see detailed compatibility][13] |
> | Cloud Provider                | AWS, Azure (preview)        | AWS, Azure, GCP, on-prem,... |
> | Operating System              | Linux                       | Linux, Windows               |
> | Serverless                    | AWS Lambda                  | -                            |
> | Container Registries          | AWS ECR (preview)           | -                            |
> 
> If you need any assistance, check our [Troubleshooting guide][14] or reach out to support@datadog.com.

Reviewer 2: ```suggestion
Use this table to decide which solution to start with: 
| Feature                                   | Agentless                                     | Unified Datadog Agent          |
|-------------------------------------------|-----------------------------------------------|--------------------------------|
| Time to deploy across your infrastructure | Minutes                                       | Hours to weeks                 |
| Vulnerability prioritization              | Yes                                           | Yes, with runtime context      |
| Vulnerability scanning frequency          | 12 hours                                      | Real-time                      |
| Vulnerability detection scope             | -                                             | -                              |
| Host and Host image                       | OS packages and App packages, mapped to image | OS packages                    |
| Container image                           | OS packages and App packages, mapped to image | OS packages                    |
| Cloud Provider                            | AWS, Azure (preview)                          | AWS, Azure, GCP, on-prem, etc. |
| Operating System                          | Linux                                         | Linux, Windows                 |
| Serverless                                | AWS Lambda                                    | -                              |
| Container Registries                      | Amazon ECR (preview)                          | -                              |

For more information on compatibility, see [CSM Vulnerabilities Hosts and Containers Compatibility][13]. If you need any assistance, see the [troubleshooting guide][14], or reach out to support@datadoghq.com.
```
